content/management/configuration-api/v3.3/index.mdx

```diff
@@ -680,6 +680,291 @@ curl -X POST \
+            "country": "United States",
+            "country_code": "US"
+          }
+        ]
+      },
+      "customer_domain": {
+        "values": [
+          {
+            "value": "livechat.com",
+            "exact_match": true
+          }
+        ]
+      }
+    },
+    "next_id": "pqi8oasdjahuakndw9nsad9na"
+  }
+]
+```
+
+</CodeResponse>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Delete Auto Access
+
+Deletes an existing `auto access` data structure specified by its ID.
+
+#### Specifics
+
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/delete_auto_access`       |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                         |
+| ---------------------- | ------------ | --------- | ----------------------------- |
+| `id`                   | Yes          | `string`  | Auto access ID                |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.3/configuration/action/delete_auto_access \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "id": "pqi8oasdjahuakndw9nsad9na"
+  }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Reorder Auto Access
+
+Moves an existing `auto access` data structure, specified by `id`, in front of another, specified by `next_id`. 
+If `next_id` is empty, then the auto access will be placed at the end.
+
+#### Specifics
+
+|                |                                                                                  |
+| -------------- | -------------------------------------------------------------------------------- |
+| Method URL     | `https://api.livechatinc.com/v3.3/configuration/action/reorder_auto_access`      |
+| Required scopes| `access_rules:rw`                                                                |
+
+
+| Parameter              | Required     | Data type | Notes                                                                                                                     |
+| ---------------------- | ------------ | --------- | ------------------------------------------------------------------------------------------------------------------------- |
+| `id`                   | Yes          | `string`  | ID of the auto access to move                                                                                             |
+| `next_id`              | No           | `string`  | ID of the auto access which should follow the moved auto access. Leave empty if the moved auto access should be the last. |
```

Comment:
(...) should be the last one on the list.
(...) should be at the end of the list.
(...) should be the last one.

It just feels like this sentence is missing something.

Reply:
Rewritten using `should be the last one on the list`